### PR TITLE
fix: all documented events have been deleted, remove text content

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

File: files/en-us/web/events/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec-urls: https://html.spec.whatwg.org/multipage/indices.html#events-2
 
 Each event is represented by an object that is based on the {{domxref("Event")}} interface, and may have additional custom fields and/or functions to provide information about what happened. The documentation for every event has a table (near the top) that includes a link to the associated event interface, and other relevant information. A full list of the different event types is given in [Event > Interfaces based on Event](/en-US/docs/Web/API/Event#introduction).
 
-This topic provides an index to the main _sorts_ of events you might be interested in (animation, clipboard, workers etc.) along with the main classes that implement those sorts of events. At the end is a flat list of all documented events.
+This topic provides an index to the main _sorts_ of events you might be interested in (animation, clipboard, workers etc.) along with the main classes that implement those sorts of events.
 
 > **Note:** This page lists many of the most common events you'll come across on the web. If you are searching for an event that isn't listed here, try searching for its name, topic area, or associated specification on the rest of MDN.
```
Comment (PassionPenguin) [inline files/en-us/web/events/index.md:15]:
```suggestion
```

**Review by PassionPenguin — COMMENTED** [review]
calls for a removal for this note, as it's also written here for the **deleted events list** (#31220)

**Comment by fuchunhui** [comment]
@yin1999 review

**Comment by yin1999** [comment]
> @yin1999 review

I am not responsible for reviewing the content under mdn/content. Please wait for the response from the assigned reviewer.

**Comment by fuchunhui** [comment]
> > @yin1999 review
> 
> I am not responsible for reviewing the content under mdn/content. Please wait for the response from the assigned reviewer.

thanks.

**Review by hamishwillee — APPROVED** [review]
Thanks very much.

**Pull request merged**